crates/forge_analyzer/src/worklist.rs

```diff
@@ -75,15 +75,20 @@ where
 
 impl WorkList<DefId, BasicBlockId> {
     #[inline]
-    pub(crate) fn push_front_blocks(&mut self, env: &Environment, def: DefId) -> bool {
+    pub(crate) fn push_front_blocks(
```

Comment:
Is adding the arguments to each worklist task still necessary, since the current checker can just access the current arguments through the callstack?